<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principal</class>
 <widget class="QWidget" name="principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Principal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="productsTable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>cod</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>category</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCreate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>470</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
